lib/output.js

```diff
@@ -148,12 +147,11 @@ exports.writeOutputs = function(outputs, baseURL, outputOpts) {
   var fileName = outFile && path.basename(outFile) || 'output.js';
 
   var output = createOutput(outFile || path.resolve(basePath, fileName), outputs, basePath, outputOpts.sourceMaps, outputOpts.sourceMapContents);
-
+	
   if (outputOpts.minify)
     output = minify(output, fileName, outputOpts.mangle, outputOpts.uglify);
 
   if (outputOpts.sourceMaps == 'inline') {
-    output.source = inlineSourceMap(output.source, output.sourceMap);
```

Comment:
This tries to inline the sourcemap, but output.sourceMap is empty when using babel, since the source map is already present in output.source (if `removeSourceMaps(source || '')` isn't called before.